.github/ISSUE_TEMPLATE/new_metapackage.yml

```diff
@@ -0,0 +1,114 @@
+      required: true
+    attributes:
+      label: Category
+      description: Which category should this tool be installed to?
+      options:
+        - Android
+        - Debuggers
+        - Delphi
+        - Developer Tools
+        - Disassemblers
+        - dotNet
+        - Flash
+        - Forensic
+        - Hex Editors
+        - Java
+        - Javascript
+        - Net
+        - Office
+        - PDF
+        - Pentest
+        - PowerShell
+        - Python
+        - Text Editors
+        - Utilities
+        - VB
+        - Web Application
+  - type: input
+    id: authors
+    attributes:
+      label: Tool's authors
+      description: Comma separated list of authors for tool
+      placeholder: ex. Elmo, Bert, Grover, Cookie Monster
+    validations:
+      required: true
+  - type: input
+    id: description
+    validations:
+      required: true
+    attributes:
+      label: Tool's description
+      description: The description should be short and not include version specific details or other information that is likely to change in a future version.
+      placeholder: ex. FakeNet-NG is a next generation dynamic network analysis tool for malware analysts and penetration testers.
+  - type: dropdown
+    id: type
+    validations:
+      required: true
+    attributes:
+      label: Package type
+      description: This is needed to know how to send the automatic PR.
+      options:
+        - METAPACKAGE
+  - type: input
+    id: dependency
+    validations:
+      required: true
+    attributes:
+      label: Dependency
+      description: Name of the package to install via dependencies.
+  - type: input
+    id: shim_path
+    validations:
+      required: true
+    attributes:
+      label: Shim path
+      description: Metapackage shim path.
+  - type: textarea
+    id: why
+    attributes:
+      label: Why is this tool a good addition?
+      description: Tell us why you would like to have this package in FLARE-VM and how this tool is useful for malware analysis, incident response, penetration testing and other security related tasking. Adding new packages involves more maintenance work, tell us what is unique about this tool that can not be done with the current toolset.
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        ---
+        We really appreciate that you share your package idea with us and we will review your issue as soon as possible. Please understand that our time is limited and we may not be able to create a package for this tool as fast as you may like. To speed up the process, please fill the issue accurately as they may allow our bot to send a PR. Note that this may not work for all cases. You can check our [Contributing Wiki page](https://github.com/mandiant/VM-Packages/wiki/Contributing) and send a pull request with the new package. We encourage the community to help us create new packages and to maintain existing ones. Join us! :hugs:
```

Comment:
```suggestion
        We really appreciate that you share your package idea with us and we will review your issue as soon as possible. Please understand that our time is limited and we may not be able to create a package for this tool as fast as you may like. To speed up the process, please fill the issue accurately as this may allow our bot to send an automatic PR. Note that this may not work for all cases. You can check our [Contributing Wiki page](https://github.com/mandiant/VM-Packages/wiki/Contributing) and send a pull request with the new package. We encourage the community to help us create new packages and to maintain existing ones. Join us! :hugs:
```